### PR TITLE
#508 Added hiding Edit button on tag list when there is no tags

### PR DESCRIPTION
**Pull request by natanielcz**

### CODE REVIEW

**Comment by dmsnell** [comment]
@natanielcz thanks again for the contribution - can you provide some context in the description here? why you want this change, screenshots or at least descriptions of the behavior you want to change?

also a quick note, we will probably not want to introduce the nested ternary, preferring instead something like explicit `if` structures or better a separate function

**Comment by natanielcz** [comment]
@dmsnell: Before my changes when we had no tags we were able to open left pane and click on `edit` button in the tags section. Then we were able to click again this time on `Done` button but actually, nothing was happening. Screenshots:
image
image

After my changes, this panel looks like this below. Edit button disappeared.
image

File: lib/tag-list.jsx [hunk 1]
```diff
@@ -53,22 +53,31 @@ export class TagList extends Component {
...
+          className="tag-list-edit-toggle button button-borderless"
+          tabIndex="0"
+          onClick={this.props.onEditTags}
+        >
+          {this.props.editingTags ? 'Done' : 'Edit'}
+        </button>
+      );
+    }
+    return null;
+  }
```
Comment (roundhill) [inline lib/tag-list.jsx]: I hate to be the 'semicolon person', but could you please add one here ;)

Comment (dmsnell) [reply lib/tag-list.jsx]: no need to @roundhill - that's what `prettier` is for :wink:

**Comment by dmsnell** [comment]
@natanielcz I was going to leave a comment then I felt like it would be annoying for you - so I hope you don't mind but I just inlined that function. if you do, please just raise it here.

**Comment by dmsnell** [comment]
@roundhill looks like this is ready to go to me!

**Comment by roundhill** [comment]
Thanks all!

**Pull request merged**